src/pages/namespace/Monitoring/Logs/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Loader2, ScrollText } from "lucide-react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "~/design/Tooltip";
+
+import { ButtonBar } from "~/design/ButtonBar";
+import CopyButton from "~/design/CopyButton";
+import ScrollContainer from "./Scrollcontainer";
+import { formatLogTime } from "~/util/helpers";
+import { useNamespacelogs } from "~/api/namespaces/query/logs";
+import { useTranslation } from "react-i18next";
+
+const LogsPanel = () => {
+  const { t } = useTranslation();
+  const { data } = useNamespacelogs();
+
+  const copyValue =
+    data?.results.map((x) => `${formatLogTime(x.t)} ${x.msg}`).join("\n") ?? "";
```

Comment:
Could "x" be named more clearly, for example "entry" if I understand correctly?

Reply:
Yes, I changed it to `logEntry` now (also in a similar code snippet in the instance logs) and will try to come up with better names for these inline function parameter :)  ca1fb2624402bfa153b6dd4c47eaa6dbc889511a